### PR TITLE
feat(): layout nav list

### PR DESCRIPTION
**Pull request by joejcarey**
## Description
moved the layout nav list into it's own branch for review 

### What's included?
- Nav list layout and route

#### Test Steps
- [x] Nav List - confirm the nav view is displaying properly. 

#### Nav List
Screen Shot 2019-03-25 at 10 02 16 AM

### CODE REVIEW

**Review by jennmedellin — APPROVED** [review]
UX LGTM

File: src/app/layouts/nav-list/nav-list.component.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  </mat-nav-list>
+  <div tdMediaToggle="gt-xs" [mediaClasses]="['push-sm']">
+    <mat-card>
+      <div layout="row" layout-align="start center">
+        <div layout="column">
+          <mat-card-title flex>Main Content</mat-card-title>
+          <mat-card-subtitle flex>content area</mat-card-subtitle>
+        </div>
+        <span flex></span>
+      </div>
```
Comment (feldominguez) [inline src/app/layouts/nav-list/nav-list.component.html]: No need for the extra 2 divs wrapping mat-card-title and subtitle. Same for the span.

**Review by feldominguez — CHANGES_REQUESTED** [review]
Linting is good now, just the extra divs on that card and we're good.

**Review by JulieMarie — APPROVED** [review]
Code LGTM

**Pull request merged**